Cue/app/tabs/library/DeckView.android.js

```diff
@@ -98,28 +98,39 @@ class DeckView extends React.Component {
       onPress: () => { this.props.navigator.pop() }
     }
 
-    const rightItems = [
+    let rightItems = [
       {
         title: 'Filter flagged cards',
         icon: this.state.isFiltering ? CueIcons.filterToggleSelected : CueIcons.filterToggle,
         onPress: () => this.setState({
           isFiltering: !this.state.isFiltering
         })
-      },
-      {
-        title: 'Add card',
-        icon: CueIcons.plus
-      },
-      {
-        title: 'Deck sharing options',
-        icon: CueIcons.share,
-      },
-      {
-        title: 'Edit',
-        icon: CueIcons.edit
-      },
+      }
     ]
 
+    if (this.props.deck.accession === 'private') {
+      rightItems = rightItems.concat([
+        {
+          title: 'Add card',
+          icon: CueIcons.plus
+        },
+        {
+          title: 'Deck sharing options',
```

Comment:
Should we do it based on whether or not there is a share code available for the deck? Because if there isn't a share code generated, should we be allowing someone that isn't the user to be generating a share code?